Resources/views/Menu/foundation_knp_menu.html.twig

```diff
@@ -82,6 +88,16 @@
     {% endif %}
 {% endblock %}
 
+{% block label %}
+    {% if item.labelAttribute('left-icon') is not empty  %}
+        <i class="{{ item.labelAttribute('left-icon') }}"></i>
```

Comment:
Can you add a entry in the documentation (readme.md) to explain how to use them ?